### ISSUE TITLE
Create a load gate for spins

### ISSUE THREAD

**Issue by fretchen** [Qiskit-Extensions/qiskit-cold-atom#20]
Currently we define the length of the spin as:

```
 job = backend.run(qc, shots = 1000, spin=Nat/2)
```

It would be much cleaner to define the spin in a similiar fashion as for the fermions.

```
circuit1.load_spins(0,Nat)
```

**Comment by eggerdj** [comment]
This will also make it easier to work with wires of different spin length.

**Comment by LaurinFischer** [comment]
Closing this issue, as this is implemented in #17. 
Note that this load instruction is not yet compatible with the simulator as this would require quite substantial changes in the numerics. 
This, however, is already discussed in another open issue #9.
